.gitignore

```diff
@@ -0,0 +1,90 @@
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
+
```

Comment:
Let's add `.DS_Store` here and then run

```
git rm -rf . --cached
git add .
```

to get rid of all remaining `DS_Store`s. Otherwise, it's good to go.